backend/inspirehep/records/marshmallow/literature/es.py

```diff
@@ -50,6 +52,27 @@ class LiteratureElasticSearchSchema(ElasticSearchBaseSchema, LiteratureRawSchema
     facet_author_name = fields.Method("get_facet_author_name")
     id_field = fields.Integer(dump_only=True, dump_to="id", attribute="control_number")
     thesis_info = fields.Nested(ThesisInfoSchemaForESV1, dump_only=True)
+    referenced_authors_bais = fields.Method(
+        "get_referenced_authors_bais", dump_only=True
+    )
+
+    @staticmethod
+    def get_referenced_authors_bais(record):
+        references_subquery = (
+            db.session.query(RecordCitations.cited_id)
+            .filter(RecordCitations.citer_id == record.id)
+            .subquery()
+        )
+        return [
+            result.author_id
+            for result in db.session.query(RecordsAuthors.author_id)
+            .filter(
+                RecordsAuthors.id_type == "INSPIRE BAI",
+                RecordsAuthors.record_id.in_(references_subquery),
```

Comment:
why not a simple join instead of the subquery?
```suggestion
                RecordsAuthors.record_id == RecordCitations.cited_id,
                RecordCitations.cited_id == record.id,
```